target/linux/realtek/dts/rtl8380_zyxel_gs1900-10hp.dts

```diff
@@ -74,3 +76,18 @@
 		};
 	};
 };
+
+&thermal_zones {
+	sfp-thermal {
+		polling-delay-passive = <10000>;
+		polling-delay = <10000>;
+		thermal-sensors = <&sfp0 &sfp1>;
```

Comment:
Nit: `<&sfp0>, <&sfp1>`

These are two sensors with 0 thermal-sensor-cells.

Reply:
now we come to the part where I didn't bother checking device tree standards. That's unnecessary. Sorry about that. Will fix.